WINDOWS/WINDOW.C

```diff
@@ -725,6 +725,9 @@ int WINAPI WinMain(HINSTANCE inst, HINSTANCE prev, LPSTR cmdline, int show)
 	    exwinmode |= WS_EX_TOPMOST;
 	if (conf_get_int(conf, CONF_sunken_edge))
 	    exwinmode |= WS_EX_CLIENTEDGE;
+#ifdef PUTTYNG
+	winmode &= ~(WS_POPUPWINDOW);
```

Comment:
I've added in c284855... the condition. Now if launched manually its normal with window title and left click menu. When hwnd_parent is present , it will be a borderless window without title